src/botconfig-example.example

```diff
@@ -93,4 +93,4 @@ const loggingInfo = {
     } as LogChannel,
 };
 // Destructing, allows for easier access to properties.
-export const { corelog, guildlog, errorlog } = loggingInfo;
+export const { corelog, guildlog, errorlog } = loggingInfo;
```

Comment:
Please revert this change (look at suggested change for reference) and change the file name back to `botconfig-example.ts` as I feel like this is not required.

```suggestion
export const { corelog, guildlog, errorlog } = loggingInfo;

```